### ISSUE TITLE
Intel MKL ERROR: Parameter 6 was incorrect on entry to DGELSD.

### ISSUE THREAD

**Issue by Arthur1252** [bubbliiiing/yolov5-pytorch#122]
Intel MKL ERROR: Parameter 6 was incorrect on entry to DGELSD.
出现了这个问题，然后Total Loss: nan || Val Loss: nan ，一直是nan，upup，这该怎么解决
![2024-03-25 09 42 16](https://github.com/bubbliiiing/yolov5-pytorch/assets/143987405/b3ed082b-09f8-4294-9370-9e14098c03ce)

**Comment by Wangxin-11** [comment]
请问你解决了吗
